common/ledger/blockledger/fileledger/factory_test.go

```diff
@@ -77,6 +88,9 @@ func TestMultiReinitialization(t *testing.T) {
 	require.Equal(t, 3, len(f.ChannelIDs()), "Expected channel to be recovered")
 	f.Close()
 
+	_, err = os.Create(filepath.Join(dir, "remove", "bar2.remove"))
+	require.NoError(t, err, "Error creating temp file: %s", err)
+
```

Comment:
This is not enough. Create a `bar2` channel dir as well, and see that the failed `Drop()` actually is repeated after restart. The end result should be that both the channel dir `bar2` and the `bar2.remove` marker file are removed.

Reply:
🧹